src/entities/observation-entity.ts

```diff
@@ -32,12 +33,13 @@ import {
   Status,
   PullusAge,
   AccuracyOfPullusAge,
-  Condition,
+  Conditions,
```

Comment:
Shouldn't be plural

Reply:
Need discuss

Reply:
https://github.com/thiagobustamante/typescript-rest-swagger/issues/27 as in this issue, Condition class name doesn't compile properly.

Reply:
Please, add comment to entity

Reply:
Done